### PR TITLE
Fix random GUI scene option widgets reset

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: pynodegl-utils/pynodegl_utils/resourcetracker.py [hunk 1]
```diff
@@ -37,7 +39,7 @@ def __init__(self):
         self._pysysdir = op.realpath(sysconfig.get_paths()['stdlib'])
 
     def _register_file(self, file):
-        if not op.isfile(file):
+        if not op.isfile(file) or op.realpath(file) == op.realpath(Config.FILEPATH):
```
Comment (mbouron-gpsw) [inline pynodegl-utils/pynodegl_utils/resourcetracker.py:42]: nit: store op.realpath(file) and reuse it for `self.filelist.update([op.realpath(file)])`

**Comment by mbouron-gpsw** [comment]
In the commit description of `utils/resourcetracker: blacklist config file`, can you mention that the 1s timeout config slot can be concurrent with query_inplace  because query_inplace() is executed from multiple QThread (used by the HooksController) ?

**Review by mbouron-gpsw — APPROVED** [review]
Other than my 2 remarks, LGTM.

File: pynodegl-utils/pynodegl_utils/resourcetracker.py (hunk 1 above)
Comment (cboesch-gpsw) [reply pynodegl-utils/pynodegl_utils/resourcetracker.py]: Code reworked

**Comment by cboesch-gpsw** [comment]
> In the commit description of `utils/resourcetracker: blacklist config file`, can you mention that the 1s timeout config slot can be concurrent with query_inplace because query_inplace() is executed from multiple QThread (used by the HooksController) ?

Commit description expanded.

**Comment by mbouron-gpsw** [comment]
> > In the commit description of `utils/resourcetracker: blacklist config file`, can you mention that the 1s timeout config slot can be concurrent with query_inplace because query_inplace() is executed from multiple QThread (used by the HooksController) ?
> 
> Commit description expanded.

LGTM, thanks.

**Pull request merged**